pkg/kubelet/cm/deviceplugin/manager_test.go

```diff
@@ -498,7 +498,6 @@ type TestResource struct {
 func TestPodContainerDeviceAllocation(t *testing.T) {
 	flag.Set("alsologtostderr", fmt.Sprintf("%t", true))
 	var logLevel string
-	flag.StringVar(&logLevel, "logLevel", "4", "test")
 	flag.Lookup("v").Value.Set(logLevel)
```

Comment:
What is this doing in the `TestPodContainerDeviceAllocation` function why is it not in the `init` function ?

Reply:
Yeah, should not be here. Thats why removed it.

Reply:
Probably remove the next line then :)